apps/docs/src/components/Pre.tsx

```diff
@@ -1,7 +1,7 @@
 import { isValidElement } from "react";
 import { Pre as NextraPre } from "nextra/components";
 
-import { Live } from "./Live";
+import { CodeBlock } from "./code/CodeBlock";
```

Comment:
do we really need to rename this? we're losing the git history, and it's harder to understand what changed